### ISSUE TITLE
The sample address is not accessible

### ISSUE THREAD

**Issue by LimaCxu** [etingof/pysnmp#389]
http://snmplabs.com/pysnmp/index.html
The sample address is not accessible

**Comment by jeffsilverm** [comment]
That's true, and it's not just a problem with my resolver.  Google's nameserver has similar issues:
```
venv) jeffs@jeffs-desktop:~/python/thumbnailer_project$ dig @8.8.8.8 snmplabs.com

; <<>> DiG 9.16.1-Ubuntu <<>> @8.8.8.8 snmplabs.com
; (1 server found)
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NXDOMAIN, id: 9053
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;snmplabs.com.			IN	A

;; AUTHORITY SECTION:
com.			816	IN	SOA	a.gtld-servers.net. nstld.verisign-grs.com. 1610320130 1800 900 604800 86400

;; Query time: 15 msec
;; SERVER: 8.8.8.8#53(8.8.8.8)
;; WHEN: Sun Jan 10 15:10:31 PST 2021
;; MSG SIZE  rcvd: 114

(venv) jeffs@jeffs-desktop:~/python/thumbnailer_project$ 
```
That NXDOMAIN answer is a major problem.
Fortunately, [the wayback machine](http://web.archive.org/web/20200221075243/http://snmplabs.com/) walked the site in February 2020, so the content is still available, at least as of a year ago.  However, the public snmplabs.com TRAP sink is not going to work.

**Comment by projx** [comment]
The developer has been experiencing real-life issues,  my thoughts are with him. 

Some users have put some temporary resources together:

Documentation - https://pysnmp.readthedocs.io
MIBS - https://github.com/projx/snmp-mibs
MibCompiler Pulls: https://raw.githubusercontent.com/projx/snmp-mibs/master/

Update:

There is also a fork, that looks like its being maintained here:

https://github.com/inexio/pysnmp

**Comment by jhofmueller** [comment]
This is sad to read :cry: any idea if this project will be receiving any attention in the future?  The fork has more recent updates but pypy still refers to this project here.

**Comment by lextm** [comment]
You can keep an eye on https://github.com/etingof/pysnmp/issues/429 and https://pysnmp.com.
